tests/testthat/test-base-plot.R

```diff
@@ -308,6 +308,31 @@ describe("Base plots", {
     vdiffr::expect_doppelganger("Character interpretation of numeric group_level", plt2)
 
   })
+
+  it("Change CI interval format [PMF-PLOT-025]", {
+    skip_on_cran()
+    skip_vdiffr()
+    plt <- plot_forest(data = sumData,
+                       CI_bracket_open = "[",
+                       CI_bracket_close = ")"
+    )
+    vdiffr::expect_doppelganger("Change CI interval format - mixed", plt)
+
+    plt <- plot_forest(data = sumData,
+                       CI_bracket_open = "(",
+                       CI_bracket_close = ")"
+    )
+    vdiffr::expect_doppelganger("Change CI interval format - change both", plt)
+
+    # error
+    error_msg <- capture_error(
+      plot_forest(data = sumData, CI_bracket_open = "]", CI_bracket_close = "]")
+    )
+    expect_true(
```

Comment:
Could this be simplified by using `expect_error`?

Reply:
I couldnt get that to work because the error message used the weird curly quotes. I could grab a section of the error message, but the curly quotes captured the main part I was looking for. Happy to adjust if you see a better approach

Reply:
I couldnt get that to work because the error message used the weird curly quotes. I could grab a section of the error message, but the curly quotes captured the main part I was looking for. Happy to adjust if you see a better approach

> * it only adds one argument rather than two, which is nice given how conceptually coupled these values are
> * it takes care of potential future requests (e.g., "I'd like to use just a space instead of a comma")
> * it uses a familiar syntax

I actually went through a bunch of options when deciding what would make the most sense. I asked @andersone1, @michaelmcd18, and @graceannobrien and settled on an approach that they liked. I like your suggestion, but as someone who doesn't _use_ the package all too much, i'd prefer to get their opinion on it (Or a scientist  - was told Katherine might be a good resource). If it's desirable to them or they dont have a strong opinion, i'd vote to do something like you're suggesting (mainly to give user's more options with fewer arguments). I am a little worried some scientists wouldn't like to specify it this way though.

In terms of deprecating `digits`, i'd like to know more about what you're envisioning down the road. As you note, the package is early in its development. Given that does it make sense to introduce kludge to support older behavior; and if so, how long (or number of versions) do we do this for? Seems like we'd remove it at some point, but just curious what the standard is for that.

Reply:
Hmm, I'm not sure what you're referring to.  Fwiw this worked fine on my end:

```diff
diff --git a/tests/testthat/test-base-plot.R b/tests/testthat/test-base-plot.R
index 98cdf0d..c627037 100644
--- a/tests/testthat/test-base-plot.R
+++ b/tests/testthat/test-base-plot.R
@@ -325,13 +325,10 @@ describe("Base plots", {
     vdiffr::expect_doppelganger("Change CI interval format - change both", plt)

     # error
-    error_msg <- capture_error(
-      plot_forest(data = sumData, CI_bracket_open = "]", CI_bracket_close = "]")
-    )
-    expect_true(
-      grepl("'arg' should be one of", error_msg$message)
+    expect_error(
+      plot_forest(data = sumData, CI_bracket_open = "]", CI_bracket_close = "]"),
+      regexp = "'arg' should be one of"
     )
-
   })
 })
```